www/jobs.md

```diff
@@ -4,20 +4,15 @@ The Hail team's mission is to build tools to enable rapid analysis and explorati
 
 We’re looking for senior engineers with experience working on large, complex projects. You don’t need experience in biology or our particular technologies, but you must be excited to be challenged and learn new things. We work in a highly multi-disciplinary environment with biologists, bioinformaticians, doctors, mathematicians, and operations.
 
-Questions? Write us at <a href="mailto:hail@broadinstitute.org"><code>hail@broadinstitute.org</code></a>. For more on the roles and **to apply, visit the [offical job post](https://broadinstitute.wd1.myworkdayjobs.com/broad_institute/job/Cambridge-MA/Software-Engineer---Hail-Team_3168)**. For more on our mission and approach, read on.
+We're specifically seeking engineers to fill the following roles:
 
-![](https://storage.googleapis.com/hail-common/hail_team.JPG "Hail team")
-
-### Why Hail? Why now?
-
-Like particle physics, astronomy, and tech before, biology has firmly entered the [fourth paradigm](https://www.microsoft.com/en-us/research/publication/fourth-paradigm-data-intensive-scientific-discovery/) of **data-intensive science** in which we measure everything and run computational experiments on the data. Genetic datasets for disease [association studies](https://en.wikipedia.org/wiki/Genome-wide_association_study) now run in the tens of terabytes, doubling every eight months. RNA-sequencing datasets measuring gene expression at single-cell resolution are measured in gigabytes but doubling far faster in the quest for a [Human Cell Atlas](https://www.broadinstitute.org/research-highlights-human-cell-atlas). Much of this data comes from the [Broad Genomics Platform](https://www.broadinstitute.org/genomics), the largest producer of human genomics information in the world.
+- Machine Learning engineer with experience in deep learning frameworks (e.g. TensorFlow) and deep learning models. Experience in Bayesian methods or with RNAseq data analysis is a plus. 
+- Front-End software engineer with a focus on scientific data visualization. Experience with data science tools in Python or R and with JS visualization technologies like D3 is desired. Bonus if you know React, Redux, node and deployment tools (Docker, k8s). You will be the Hail team's first front-end person, and will have the opportunity to set the vision.
+- Distributed Systems engineer who will help design the architecture to allow us to derive scientific insights from petabyte-scale datasets now and scale 3x/year for the foreseeable future.
```

Comment:
For these three (which do not have workday links yet, I feel like we should do something like "if you're interested [email us](mailto:hail@broadinstitute.org)"

Reply:
I think the link you suggested removing above is the generic job app link, so I could just link to that for these three instead.